test/StandardSqlFormatterTest.js

```diff
@@ -202,6 +202,66 @@ describe("StandardSqlFormatter", function() {
         );
     });
 
+    it("formats simple SELECT", function() {
+        const result = sqlFormatter.format("SELECT 'value'", );
+        expect(result).toBe(
+            "SELECT\n" +
+            "  'value'\n"
+        );
+    });
+
+    it("formats simple SELECT with national character (MSSQL)", function() {
+        const result = sqlFormatter.format(
+            "SELECT N'value';"
+        );
+        expect(result).toBe(
+            "SELECT\n" +
+            "  N'value';\n"
+        );
+    });
+
+    it("formats SELECT with complex WHERE", function() {
```

Comment:
This test file is getting quite huge. We could save some lines when merge this test and below one:

```
it("formats SELECT with complex WHERE", function() {
    const result = sqlFormatter.format(
        "SELECT * FROM foo WHERE Column1 = N'testing' AND Column2 = 'testing'" +
        "AND ( (Column3 = Column4 OR Column5 >= NOW()) );"
    );
    expect(result).toBe(
        "SELECT\n" +
        "  *\n" +
        "FROM\n" +
        "  foo\n" +
        "WHERE\n" +
        "  Column1 = N'testing'\n" +
        "  AND (\n" +
        "  Column2 = 'testing'\n" +
        "    (\n" +
        "      Column3 = Column4\n" +
        "      OR Column5 >= NOW()\n" +
        "    )\n" +
        "  );\n"
    );
});
```